pom.xml

```diff
@@ -1730,7 +1730,10 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-shade-plugin</artifactId>
-                    <version>3.2.4</version>
+                    <version>3.4.1</version>
+                    <configuration>
+                        <createDependencyReducedPom>false</createDependencyReducedPom>
+                    </configuration>
```

Comment:
any reason we are setting this globally instead of defining it where we actually use the plugin?

Reply:
i wasn't able to figure out where the plugin was being used from the error messages I was seeing so i set it globally, the way you did it in the other pr was better

Reply:
thanks, your PR actually helped me figure out why my build was failing initially. I had missed one of the uses of the plugin which was causing cryptic error messages.